bitcoin/src/blockdata/script/push_bytes.rs

```diff
@@ -111,6 +111,8 @@ mod primitive {
         }
     }
 
+    // TODO add 2024 edition ranges when available
+    // see https://rust-lang.github.io/rfcs/3550-new-range.html
```

Comment:
I don't think we will remember this comment. An issue would be mildly better but really, we should add it to our calendars. :)

Reply:
Right, I will make an issue so others see it as well

Reply:
When gathering some resources to mention in the issue I couldn't find the range thing [here](https://doc.rust-lang.org/edition-guide/rust-2024/index.html), some further searching got me [here](https://github.com/rust-lang/rust/issues/123741#issuecomment-2255133652), the new range types likely won't make it in the 2024 edition.

Reply:
That's sad but anyway, I think we don't have to open an issue now since we'll have plenty of time to do that.